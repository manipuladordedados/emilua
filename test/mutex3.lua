-- Deadlock detection

local mutex = require('mutex')

m = mutex.new()
m:lock()
m:lock()
