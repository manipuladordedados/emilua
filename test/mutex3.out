Possible deadlock(s) detected during VM 0x1 shutdown:
	No scheduled fibers remaining to unlock mutex 0x2
